rust/sbp2json/src/bin/sbp2json.rs

```diff
@@ -46,8 +59,15 @@ fn main() -> Result<()> {
 
     env_logger::init();
 
-    let stdin = io::stdin();
-    let stdout = io::stdout();
+    let stdin: Box<dyn Read> = match options.input {
+        Some(path) => Box::new(File::open(path).unwrap()),
```

Comment:
Should be able to change these `unwrap()`s to `?`s which will print a nicer error message